Jenkinsfile

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env groovy
+
+/**
+ * Execute block and set GitHub commit status to success or failure if block
+ * throws an exception.
+ *
+ * @param label The context for the commit status.
+ * @param block The block to execute.
+ */
+def withCommitStatus(label, block) {
+  try {
+    // Execute steps in stage
+    block()
+
+    currentBuild.result = 'SUCCESS'
+  } catch(error) {
+    currentBuild.result = 'FAILURE'
+    throw error
+  } finally {
+
+    // Mark commit with final status
+    step([ $class: 'GitHubCommitStatusSetter'
+         , contextSource: [$class: 'ManuallyEnteredCommitContextSource', context: "Velocity " + label]
+         ])
+  }
+}
+
+/**
+ * Wrap block with a stage and a GitHub commit status setter.
+ *
+ * @param label The label for the stage and commit status context.
+ * @param block The block to execute in stage.
+ */
+def stageWithCommitStatus(label, block) {
+  stage(label) { withCommitStatus(label, block) }
+}
+
+node('JenkinsMarathonCI-Debian8') {
+    try {
+        stage("Checkout Repo") {
+            checkout scm
+            gitCommit = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+            shortCommit = gitCommit.take(8)
+            currentBuild.displayName = "#${env.BUILD_NUMBER}: ${shortCommit}"
+        }
+        stage("Install Mesos") {
+            sh "sudo apt-get -y clean"
+            sh "sudo apt-get -y update"
+            sh "sudo apt-get install -y --force-yes --no-install-recommends curl"
+            sh """if grep -q MesosDebian \$WORKSPACE/project/Dependencies.scala; then
```

Comment:
Good point. We could define a matrix with different versions in the future. However, this resembles what we have right now.